topdown/eval.go

```diff
@@ -1555,11 +1553,10 @@ func (e *eval) resolveReadFromStorage(ref ast.Ref, a ast.Value) (ast.Value, erro
 	}
 
 	v, err := e.external.Resolve(e, ref)
-
 	if err != nil {
 		return nil, err
-	} else if v == nil {
-
+	}
+	if v == nil {
 		path, err := storage.NewPathForRef(ref)
 		if err != nil {
```

Comment:
☝️ the stuff above is of cosmetic nature, the relevant change is below 👇

Reply:
Updated the PR: the conversion is now done in the inmem store, and it'll return a *ast.lazyObj, but that flies through here as ast.Value.